test/appium/support/testrail_report.py

```diff
@@ -0,0 +1,68 @@
+import json
+import requests
+import base64
+from os import environ
+from support.base_test_report import BaseTestReport
+
+
+class TestrailReport(BaseTestReport):
+
+    def __init__(self, sauce_username, sauce_access_key):
+        super(TestrailReport, self).__init__(sauce_username, sauce_access_key)
+
+        self.password = environ.get('TESTRAIL_PASS')
+        self.user = environ.get('TESTRAIL_USER')
+
+        self.run_id = None
+        self.suite_id = 42
+        self.project_id = 9
+
+        self.outcomes = {
+            'passed': 1,
+            'undefined_fail': 10}
+
+        self.headers = dict()
+        self.headers['Authorization'] = 'Basic %s' % str(
+            base64.b64encode(bytes('%s:%s' % (self.user, self.password), 'utf-8')), 'ascii').strip()
+        self.headers['Content-Type'] = 'application/json'
+
+        self.url = 'https://ethstatus.testrail.net/index.php?/api/v2/'
```

Comment:
Those all (`run_id`, `suite_id`, `project_id`, `url`, ..) should probably be constants. Like `RUN_ID`